traitlets/config/application.py

```diff
@@ -422,7 +422,7 @@ def print_help(self, classes=False):
         self.print_options()
 
         if classes:
-            help_classes = self._classes_with_config_traits()
+            help_classes = self.classes
```

Comment:
Sorry, now I saw the change in loader below, so you really mean that :-)
But isn't that a breaking change?